### PR TITLE
Bai/1148 unit test for review role display

### PR DESCRIPTION
**Pull request by TT38665**

### CODE REVIEW

File: frontend/src/reviews/ReviewRoleDisplay.spec.tsx
Comment (ARADDCC012) [inline frontend/src/reviews/ReviewRoleDisplay.spec.tsx:25]: This highlights that we should update the component so that the message says "access request" instead of just "access". Not for you to do here though :)

File: frontend/src/reviews/ReviewRoleDisplay.tsx
Comment (ARADDCC012) [inline frontend/src/reviews/ReviewRoleDisplay.tsx:28]: Looks like this `data-test` isn't being used in your tests so it can be removed.

**Review by ARADDCC012 — CHANGES_REQUESTED** [review]

**Review by ARADDCC012 — APPROVED** [review]

**Comment by ARADDCC002** [comment]
I think before we merge this in we wait until my v2 testing PR is merged in, and then we can update this one so that the test file is in the new testing directory. Happy to sort that for you though.

**Review by ARADDCC012 — CHANGES_REQUESTED** [review]
Hold off on merging until https://github.com/gchq/Bailo/pull/1089 has been merged into main

**Pull request closed without merging**
